### PR TITLE
fixed  airbnb style guide errors and some other fixes

### PR DESCRIPTION
**Pull request by connieelee**
- fixed almost all errors in console (expect one, but that will be fixed in time. I left a comment where  it's supposed to be)
- fixed almost all airbnb style guide issues (expect the bind on prop methods...)
- updated MUI css cdn link (for whatever reason they changed their url. maybe we should npm install as a dependency eventually to ensure nothing goes wrong in the future)
- dbdefs populate into their rightful slots, and "save" button is disabled unless a change is made (but save functionality is not implemented yet

_** VERY IMPORTANT NOTE: **_
Changed the app's INDEX.JS file to INDEX.JSX because we are not supposed to have jsx in a file with a .js extension. This will cause your `npm start` to break. Here's what you have to do --
1) Search for `appIndexJs` and open up `Sushi/node_modules/react-scripts/config/paths.js` 
2) There are two lines (lines 43 and 62 on my file) that read `appIndexJs: resolveApp('src/index.js'),`. Change `index.js` in both lines to `index.jsx`and it should work!

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,47 @@
+0 info it worked if it ends with ok
```
Comment (Abazhenov) [inline npm-debug.log]: Should we .gitignore this file?

Comment (clocasto) [inline npm-debug.log]: I think so, yeah.

**Comment by connieelee** [comment]
Ok, finally resolved the conflicts. Sorry it's because I didn't merge from master over the weekend (oops).

**Comment by connieelee** [comment]
Also ignore the very important note that turned out to be not very important (sorry again guys)

**Pull request merged**